assets/dev/js/editor/controls/behaviors/scrubbing.js

```diff
@@ -0,0 +1,134 @@
+const UPDATE_VALUE = 'UPDATE-VALUE';
+const UPDATE_VALUE_ENHANCED = 'UPDATE-VALUE-ENHANCED';
+
+module.exports = class FooterSaver extends Marionette.Behavior {
+	intentVerified = false;
+
+	checkIntentTimeout = null;
+
+	intentTime = 600;
+
+	skipperCount = 0;
+
+	skipperMax = 10;
+
+	enhancedNumber = 10;
+
+	ui() {
+		return {
+			input: 'input',
+			label: 'label',
+		};
+	}
+
+	events() {
+		return {
+			'mousedown @ui.input': 'onMouseDownInput',
+			'mousedown @ui.label': 'onMouseDownLabel',
+			'mouseover @ui.label': 'onMouseOverLabel',
+		};
+	}
+
+	scrub( input, movementEvent ) {
+		// Determine the updating rhythm.
+		const movementType = this.checkKeyDown( movementEvent );
+
+		if ( ! movementType ) {
+			return;
+		}
+
+		// Actually change the input by scrubbing.
+		switch ( movementType ) {
+			case UPDATE_VALUE:
+				input.value = +input.value + movementEvent.movementX;
+				break;
+
+			case UPDATE_VALUE_ENHANCED:
+				input.value = +input.value + ( movementEvent.movementX * this.enhancedNumber );
+				break;
+
+			default:
+				break;
+		}
+
+		// Fire inputEvent after changing input data, for validation purposes.
+		input.dispatchEvent( new Event( 'input', { bubbles: true } ) );
+	}
+
+	checkKeyDown( movementEvent ) {
+		if ( movementEvent.altKey && ! movementEvent.ctrlKey ) {
+			this.skipperCount++;
+			// When ALT key is pressed, skipping x times before updating input value.
+			if ( this.skipperCount <= this.skipperMax ) {
+				return;
+			}
+			// Skipped x time, now (update value &) restart counting.
+			this.skipperCount = 0;
```

Comment:
I think this part can be simplified using [modulo ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Remainder)
Talk with me when you're on it.

Reply:
Modulo will work, but it doesn't make the code shorter, or faster, or clearlier.

Reply:
It'll probably remove redundant variables.
let's talk about it if you want